main.tf

```diff
@@ -17,7 +17,14 @@
 resource "random_id" "random_hash_suffix" {
   byte_length = 4
 }
+resource "null_resource" "org_id_or_folder_id_is_empty" {
+  count = "${var.org_id == "" && var.folder_id != "" ? 1 : 0}"
 
+  provisioner "local-exec" {
```

Comment:
what if folder_id == ''" and org_id is not? is it ok?

Reply:
ok, it was a typo, fixed